packages/app/src/app/pages/Sandbox/index.tsx

```diff
@@ -23,6 +23,22 @@ export const Sandbox = React.memo<Props>(
     const state = useAppState();
     const actions = useActions();
 
+    /**
+     * !important Hard bug fix
+     *
+     * This address temporarily many unexpected
+     * behaviors that occurs when  navigating away
+     * from the editor and go back to it
+     * Eg: Editor -> Dashboard -> Editor
+     *
+     * @link https://www.notion.so/Unexpected-behaviors-when-navigating-away-from-the-editor-and-go-back-to-it-b5fcc670c3fb46afa8a57dd05f701bcb
+     */
+    useEffect(() => {
+      if (window.CSEditor) {
+        window.location.reload();
+      }
+    }, []);
```

Comment:
Is this running the CodeSandbox call API twice?

Reply:
@CompuIves Yes, it seems that it calls the API twice. In order to avoid this, I tried to put this conditional in the root of the computer, however, this `reload` was triggered in the "Fork" command, which is something that we don't want. 

Apart from this is not a so nice workaround, (very well known as "gambiarra" in Portuguese), I'd say this's the best we could do now, as far as I am concerned.